### PR TITLE
Add page on sending ERC-20s via XCM

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This adds a new page to the XC-20s section of the docs on how to send ERC-20s using the X-Tokens Pallet and Precompile. It also moves away from mintable xc-20s, favoring local xc-20s instead.

~~Still need to figure out the naming situation but for now just referring to regular old ERC-20s as ERC-20s... groundbreaking, I know.~~

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira

### After Translation Requirements

- [x] No additional PRs are required after the translations are done

### CODE REVIEW

**Comment by albertov19** [comment]
Hey because this is still being worked on right now, let's convert this into a draft

**Comment by eshaben** [comment]
Once other XCM pages are updated to XCM v3, I'll circle back to update the `send-erc20s.md` file to include these snippets: 'text/xcm/xcm-precompile-multilocation.md' and 'text/xc-20/xtokens/multilocation-structures.md'. Didn't want to go down the rabbit hole of updating these snippets and the pages they are on in this PR

**Comment by eshaben** [comment]
@albertov19 this can be reviewed. I only updated the new page to reflect changes to XCM v3. I've been making changes to other related pages in another branch, so I'll open another PR for those pages 🙂

File: builders/interoperability/xcm/xc20/.pages
Comment (eshaben) [inline builders/interoperability/xcm/xc20/.pages:7]: moved this to the bottom as it doesn't follow the same format as the 'External XC-20s' page and more aligns with the 'Send External XC-20s' page. Not sure about the name, but I did want to make it clear for those searching that this page is related to sending ERC-20s (for SEO purposes)

File: builders/interoperability/xcm/xc20/xcm-erc20s.md
Comment (eshaben) [inline builders/interoperability/xcm/xc20/xcm-erc20s.md:70]: I've been thinking about this and I think tomorrow I'm going to remove these tables. It made sense to have tables like these when using Polkadot.js Apps, but now I think it just makes more sense to focus on the JavaScript code

**Comment by eshaben** [comment]
With all the changes I ended up moving the xtokens v3 update into this PR 😅

File: builders/interoperability/xcm/xc20/overview.md
Comment (eshaben) [inline builders/interoperability/xcm/xc20/overview.md:287]: I know we talked about having this here, but not sure it really makes sense 🤷‍♀️

**Review by albertov19 — APPROVED** [review]
LGTM! Let's doo iitttttt!

**Comment by albertov19** [comment]
@eshaben I've approved the PR, please check my commits just to check everything looks good :)

File: .snippets/text/xcm/xcm-precompile-multilocation.md
Comment (eshaben) [inline .snippets/text/xcm/xcm-precompile-multilocation.md:46]:
```suggestion
|  AccountId32   | "0x01+AccountId32+02"  |   AccountId32, Network Polkadot    |
```

Polkadot should be 2

File: builders/interoperability/xcm/xc20/overview.md
Comment (eshaben) [inline builders/interoperability/xcm/xc20/overview.md:16]:
```suggestion
XC-20s are a unique asset class on Moonbeam. It combines the power of Substrate assets (native interoperability) but allows users and developers to interact with them through a familiar [ERC-20 interface](#the-erc20-interface){target=_blank}. On the EVM side, XC-20s have an [ERC-20 interface](#the-erc20-interface){target=_blank}, so smart contracts and users can easily interact with them, and no knowledge of Substrate is required. This ultimately provides greater flexibility for developers when working with these types of assets and allows seamless integrations with EVM-based smart contracts such as DEXs and lending platforms, among others. Moreover, developers can integrate XC-20s with regular [Ethereum development frameworks](/builders/build/eth-api/dev-env/){target=_blank} or dApps, and create connected contracts strategies with such assets. Moreover, with the introduction of [RT2301](https://github.com/PureStake/moonbeam/tree/runtime-2301){target=_blank}, all ERC-20s are XCM-ready, meaning they can also be referred to as XC-20s.
```

File: builders/interoperability/xcm/xc20/overview.md
Comment (eshaben) [inline builders/interoperability/xcm/xc20/overview.md:39]:
```suggestion
The main difference between these two types of assets is that local XC-20s are EVM ERC-20s that have XCM capabilities, while external XC-20s are Substrate assets with an ERC-20 interface on top.
```

File: builders/interoperability/xcm/xc20/overview.md
Comment (eshaben) [inline builders/interoperability/xcm/xc20/overview.md:47]:
```suggestion
**Local XC-20s that are registered on other chains must comply with the standard ERC-20 interface as described in [EIP-20](https://eips.ethereum.org/EIPS/eip-20){target=_blank}.**
```

File: builders/interoperability/xcm/xc20/overview.md
Comment (eshaben) [inline builders/interoperability/xcm/xc20/overview.md:78]:
```suggestion
The current list of available external XC-20 assets per network is as follows:
```

File: builders/interoperability/xcm/xc20/overview.md
Comment (eshaben) [inline builders/interoperability/xcm/xc20/overview.md:167]:
```suggestion
## XC-20s Solidity Interface {: #xc20s-solidity-interface }
```

**Comment by eshaben** [comment]
This PR can be merged once this one is approved: https://github.com/PureStake/moonbeam-docs-cn/pull/276

**Pull request merged**
